services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java

```diff
@@ -275,7 +275,7 @@ public String getVersion() throws RepositoryException {
    * @param port the port
    * @return the string builder
    */
-  protected StringBuilder platformUrl(String schema, String host, int port) {
```

Comment:
It should stay protected as it's only a helper, not something service level.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/31/commits/ca043163690bc4c751e447a58539aceb6e4ce310